gateway/src/shard/processor/impl.rs

```diff
@@ -184,8 +184,8 @@ impl ReceivingEventError {
         matches!(
             self,
             ReceivingEventError::AuthorizationInvalid { .. }
-            | ReceivingEventError::IntentsDisallowed { .. }
```

Comment:
Where do these changes come from? rustfmt, on nightly? We use stable, and I can't reproduce so this should probably be kept as-is.